WBooks/Books_Table/BooksTableViewController.swift

```diff
@@ -50,13 +50,13 @@ class BooksTableViewController: UIViewController {
         
         _view.tableView.register(cell: BookTableViewCell.self)
         
-        booksViewModel.loadBooks()
+        _booksViewModel.loadBooks()
         
-        booksViewModel.books.producer.startWithValues { [unowned self] _ in
+        _booksViewModel.books.producer.startWithValues { [unowned self] _ in
             self._view.tableView.reloadData()
         }
         
-        booksViewModel.errorsSignal.observeValues({ [unowned self] (error) in
+        _booksViewModel.errorsSignal.observeValues({ [unowned self] (error) in
             let alertError = UIAlertController(title: Constants.errorAlertTitle,
                                                message: error.localizedDescription,
                                                preferredStyle: .alert)
```

Comment:
You can start subdividing this function. Since this function is always doing most of the things and adding more and more, we try to subdivide it. For example, in this case, you could have: 
```
override func viewDidLoad() {
    super.viewDidLoad()
    configureTable(_view.tableView)
    bindViewModel(_booksViewModel)
    _booksViewModel.loadBooks()
}
```